app/models/course/condition/assessment.rb

```diff
@@ -40,6 +45,10 @@ def self.dependent_class
     Course::Assessment.name
   end
 
+  def self.on_dependent_status_change(submission)
+    resolve_conditional_for(submission.course_user) if submission.graded?
```

Comment:
Yup it is.

Should we then implicitly consider a nil minimum grade percentage as the condition that "submitted is sufficient"?